src/TableContainer.js

```diff
@@ -30,6 +31,20 @@ function TableContainer({
     queryString = queryString.replace('"desc"', 'desc')
   }
 
+  const newFilter = {}
+
+  if (filter && filter.where) Object.assign(newFilter, { where: filter.where })
+
+  const getTotalRows = new Query(`${dataset}_aggregate`)
+    .filter(newFilter)
+    .find(new Query('aggregate').find('count'))
+
+  // getTotalRows.filter({where: {ConnectedArea: {_eq: 'DK1'}}});
+  const TOTALQUERY = gql`
```

Comment:
@cotts it cannot be here because `TableContainer` has nothing to do with total rows.